### ISSUE TITLE
Day Planner font color

### ISSUE THREAD

**Issue by Owlf-Dev** [deathau/80s-Neon-for-Obsidian.md#18]
This is such a cool theme. Would be great if the font color could be white when using the Day Planner plugin. The font color is currently black on the tasks which makes it really hard to see the tasks given the lack of color contrast.

**Comment by alegmalinovsky** [comment]
changes ive made to address similar issue were merged (https://github.com/deathau/80s-Neon-for-Obsidian.md/pull/25), please make sure you're on the latest version and check again
